src/routes/onboarding/set-medication-reminders/medication-section.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Form, Button } from "../../../components"
+import { faTrashCan } from "@fortawesome/free-regular-svg-icons";
+
+interface Props {
+  index?: number;
+  onDelete?: (index:number) => void;
+  elementId?: number;
+}
+
+export const MedicationSection: React.FC<Props> = ({ index, onDelete, elementId}) =>  {
+  const inputName = `medication${elementId}`
+
+  type TextInputValue = Record<typeof inputName, string>;
+
+  const {
+    register,
+    unregister,
+    formState: {errors},
+  } = useForm<TextInputValue>();
+
+
+  return (
+    <div className="flex flex-col gap-5 bg-white mx-auto min-w-[315px] max-w-[400px] px-4 py-3 w-full rounded-[16px]">
+          <div className="flex items-centerw">
+            <h3 className="font-text text-section-subtext font-normal leading-[24px] text-center">Medication {index}</h3>
```

Comment:
looking at the Figma, this h3 should have a font-weight of 500 (tailwind class is `font-normal`)
also, the font size should be 24px (tailwind `text-2xl`)